services/services.go

```diff
@@ -240,7 +246,7 @@ func getRelaysPageData() (*types.RelaysResp, error) {
 			validators.validatorindex as proposer,
 			encode(exec_extra_data, 'hex') as block_extra_data
 		from (
-			select * 
+			select value, block_slot, builder_pubkey, proposer_fee_recipient, block_root, tag_id
```

Comment:
This is requiring `proposer_pubkey` as it is used for the `left join`
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4887c3412af6f61c937a9d70d2ab363c6e8324eb/services/services.go#L260
and the `group by`
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4887c3412af6f61c937a9d70d2ab363c6e8324eb/services/services.go#L264
below.

Like this:
```suggestion
			select value, block_slot, builder_pubkey, proposer_fee_recipient, block_root, tag_id, proposer_pubkey
```

Reply:
added proposer_pubkey